client_end_user/shared/src/commonMain/kotlin/presentation/search/SearchUiEffect.kt

```diff
@@ -0,0 +1,6 @@
+package presentation.search
+
+sealed class SearchUiEffect {
+    data object NavigateToMeal : SearchUiEffect()
+    data object NavigateToRestaurant : SearchUiEffect()
```

Comment:
why data object and not data class or object ?

Reply:
@Saif-Alhaider not done yet